etc/ci/bencher.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import argparse
+import json
+import os
+
+
+def size(args):
+    size = os.path.getsize(args.binary)
+    print(size)
+    with open(args.bmf_output, 'w', encoding='utf-8') as f:
+        json.dump({
+            'servo': {
```

Comment:
Can we make this configurable? 
Ideally this should include the target triple, cargo features (if not default) and maybe the cargo profile.
I think we want to track the binary size for all the different platforms, and I would also want to track `--no-default-features` vs `default-features` and also the size with different profiles, e.g. a profile with `opt-level=3` vs `opt-level=z`.

This comment is definitly not blocking though, just a suggestion that we can follow-up on zulip or in an issue.

Reply:
The problem is that we would need to do all these build in CI to actually track them for file size.

Reply:
Sure - but at least for the `main` branch we don't actually have that many new commits per day. I think it makes sense to gather more data for the main branch, since that is quite useful to at least identify large code-size increases quickly.

Spotting such issues before they are merged is of course even better, but that indeed requires quite a lot more build-time and probably would mean we would first need more build servers (or optimize caching)